proposals/poc/witx/proposal_symmetric.md

```diff
@@ -0,0 +1,1418 @@
+# Types
+## <a href="#crypto_errno" name="crypto_errno"></a> `crypto_errno`: Enum(`u16`)
+Error codes.
+
+Size: 2
+
+Alignment: 2
+
+### Variants
+- <a href="#crypto_errno.success" name="crypto_errno.success"></a> `success`
+Operation succeeded.
+
+- <a href="#crypto_errno.guest_error" name="crypto_errno.guest_error"></a> `guest_error`
+An error occurred when trying to during a conversion from a host type to a guest type.
+
+Only an internal bug can throw this error.
+
+- <a href="#crypto_errno.not_implemented" name="crypto_errno.not_implemented"></a> `not_implemented`
+The requested operation is valid, but not implemented by the host.
```

Comment:
If the host does not implement it, can the host be sure that it is valid?

Reply:
Maybe `currently not implemented` would be indeed better here.

Reply:
Changed to `currently not implemented`.